### PR TITLE
MARKET-1305  Lottie Animation

### PR DESCRIPTION
**Pull request by StasivPavlo**

### CODE REVIEW

**Review by addKiwi — COMMENTED** [review]
Don't create an external directory for 1-2 file
https://github.com/Backendless/ui-builder-library/wiki/Files#general-rules
I think we don't need a `lib` folder

File: components/bl-lottieAnimation-component/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<dd>Allows to specify looping or non-looping animation. Default value 'true'</dd>
+<dt>Stopped</dt>
+<dd>Allows to specify stopped or not stopped animation.</dd>
+</dl>
+
+### Event Handlers and Bindable Properties
+
+<dl>
+<dt>Data Logic</dt>
+<dd>Allows to specify data for animation in Backendless logic</dd>
```
Comment (v-excelsior) [inline components/bl-lottieAnimation-component/README.md:25]: Could we add a little instruction with links where user could find/create an animation?

File: components/bl-lottieAnimation-component/src/index.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    if (isStopped) {
+      stop();
+    }
+  }, [data]);
+
+  component.playAnimation = () => {
+    play();
+  };
+
+  component.stopAnimation = () => {
```
Comment (v-excelsior) [inline components/bl-lottieAnimation-component/src/index.js:24]: What if we add an action to change the current animation? I know we could do it through data binding but action also could be useful.

File: components/bl-lottieAnimation-component/src/index.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import { useEffect, useRef } from 'react';
+import { destroy, loadAnimation, play, stop } from './lottie.min';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function lottieAnimation({ component, eventHandlers }) {
+  const { display, classList, data, isLoop, type, isStopped } = component;
+  const { onHover, onUnhover } = eventHandlers;
```
Comment (v-excelsior) [inline components/bl-lottieAnimation-component/src/index.js:8]: Add onClick event please

**Review by v-excelsior — CHANGES_REQUESTED** [review]

File: components/bl-lottieAnimation-component/src/index.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
+import { useEffect, useRef } from 'react';
+import { destroy, loadAnimation, play, stop } from './lottie.min';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function lottieAnimation({ component, eventHandlers }) {
```
Comment (Valodya) [inline components/bl-lottieAnimation-component/src/index.js:6]: `LottieAnimation`

File: components/bl-lottieAnimation-component/README.md (hunk 1 above)
Comment (Valodya) [reply components/bl-lottieAnimation-component/README.md:28]: @StasivPavlo ?

**Pull request merged**